synapse/handlers/acme.py

```diff
@@ -73,7 +73,9 @@ async def start_listening(self) -> None:
                 "Listening for ACME requests on %s:%i", host, self.hs.config.acme_port
             )
             try:
-                self.reactor.listenTCP(self.hs.config.acme_port, srv, interface=host)
+                self.reactor.listenTCP(
+                    self.hs.config.acme_port, srv, backlog=50, interface=host
```

Comment:
mypy was complaining about not giving the required `backlog` attribute, 50 matches the default in all the reactor implementations in Twisted.